### DISCUSSION TITLE
v3.0 - The Big One

### DISCUSSION THREAD

**Discussion by IdreesInc** [IdreesInc/Monocraft#126] [Announcements]
Thanks to the brilliant work of @dantaeusb  and @Ciubix8513, Monocraft now has over 1500+ glyphs and continuous ligatures! I mean look at this:
<img width="856" alt="glyphs" src="https://github.com/IdreesInc/Monocraft/assets/4875804/1659b0cf-0c20-438a-ada9-7ef270da0229">

@dantaeusb has added almost 1000 characters to the glyph set, bringing the total number of characters to 1455! Most of these include characters from non-European languages and a number of these characters are not monospaced.
@Ciubix8513  has enabled the use of continuous ligatures! With her contributions, you can now type arrows and lines up to 30 characters long (which seems excessive, but hey who am I to judge?).

Since this release contains so many new characters, there's always a chance that a bug slipped through the cracks. Please report any issues through the repo's "Issues" tab and make any comments in the discussion down below. Thanks once again to all the amazing people that have built and improved this font!

## What's Changed
* Extend characters with European non-latin set by @dantaeusb in https://github.com/IdreesInc/Monocraft/pull/105
* Added continuous ligatures  by @Ciubix8513 in https://github.com/IdreesInc/Monocraft/pull/116
* V3 by @IdreesInc in https://github.com/IdreesInc/Monocraft/pull/125

## New Contributors
* @dantaeusb made their first contribution in https://github.com/IdreesInc/Monocraft/pull/105

**Full Changelog**: https://github.com/IdreesInc/Monocraft/compare/v2.5...v3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/IdreesInc/Monocraft/releases/tag/v3.0'>v3.0 - The Big One</a>.</em>

**Comment by sol-vin** [comment]
Sincerely, thank you so much for this project. It's so good, I use it as my daily driver.

**Reply by IdreesInc** [reply]
I'm glad to hear you like it! With the help of a lot of fantastic contributors, it has evolved from a little joke project to a surprisingly comprehensive font.

**Comment by Woovie** [comment]
Can the various release builds be described in a meaningful way? Why would I use the one that says "nerd" and "patched" in the name? I couldn't find any obvious label anywhere explaining differences.
Thanks!

**Reply by Cyklan** [reply]
The one without any flags in the name is the one described in this discussion, with more characters and longer ligatures.

The `no-ligatures` file should be quite obvious, it doesn't contain the ligatures.

The `nerd-font-patched` one contains a bunch of icons that are required by a lot of terminal applications. You can read more about these here: https://www.nerdfonts.com/

**Reply by Woovie** [reply]
Thanks for the reply and clarifications.

**Comment by singurvir** [comment]
Im typing Monocraft into the font family settings on vs code but its still not showing up any help perchance :(. I'm on mac

**Reply by Anthracited** [reply]
Same :( Did you find a way to get it to work?

**Comment by GG-G-M** [comment]
Very nice when used as font for VS Code
<img width="1435" height="614" alt="image" src="https://github.com/user-attachments/assets/5c245375-5187-4c86-88d4-1df044d904db" />
